pkg/csi/driver/bv_controller.go

```diff
@@ -1231,8 +1231,7 @@ func (d *BlockVolumeControllerDriver) ControllerExpandVolume(ctx context.Context
 	if newSizeInGB <= oldSize {
 		log.Infof("Existing volume size: %v Requested volume size: %v No action needed.", *volume.SizeInGBs, newSizeInGB)
 		return &csi.ControllerExpandVolumeResponse{
-			CapacityBytes:         oldSize,
-			NodeExpansionRequired: true,
```

Comment:
This needs to be retained for cases where controller container might restart after block volume expand, node driver will handle the no-op case where node expansion is already done